src/types.ts

```diff
@@ -234,6 +234,19 @@ export interface Segment {
 
 export type SegmentType = "sprint" | "climb" | "segment";
 
+export interface Bike {
+  make: string;
+  name: string;
+  year?: number;
+  level: number; // negative for special cases like Tron bike
+  frameWeight: number;
+  frameCdA: number;
```

Comment:
Some frames have 0 or a negative number as CdA. Not an aero expert, but this looks more like the difference of the current frame compared to a "default bike". Maybe we can be more explicit in the name about this. A negative bike would probably ride itself :D